pkg/controller/utils/utils.go

```diff
@@ -32,6 +31,66 @@ var (
 		"iam:SimulatePrincipalPolicy", // needed so we can verify the above list of course
 	}
 
+	CredPassthroughActions = []string{
```

Comment:
credPassthroughActions I think might be correct syntax. CredMintingActions above looks off too.